dmscripts/generate_framework_master_csv.py

```diff
@@ -85,7 +85,7 @@ def _update_with_supplier_data(self, output):
             ]
             # This creates placeholders for the dynamic lot fieldnames.
             lot_placeholders = [0 for i in self._get_dynamic_field_names()]
-            supplier_dict = dict(zip(field_names, supplier_info + lot_placeholders))
+            supplier_dict = dict(list(zip(field_names, supplier_info + lot_placeholders)))
```

Comment:
Aha, good spot. `2to3` can be overly cautious when wrapping these, I did remove a few unnecessary ones (e.g. where the iterator was only used in a for loop) but missed this one.